_posts/2019-02-22-Rich-Media-Creatives.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: post
+title: Rich Media Creatives
+subtitle: Seamless publisher integration
+category: frontend
+tags: [frontend, mobile]
+author: Cătălin Berța
+header-img: "images/Rich-Media-Creatives/header-bg.jpg"
+---
+
+As an introductory article to the frontend section of the blog, we would like to give a quick overview of some of the best practices that our creatives go through before being delivered to various publishers that come in many forms and sizes.
+
+The speed of the publisher web pages can be often hurt by 3rd party content, the publishers having somewhat no control over what the content loads on its end, how big the files are or how that impacts the end-user experience.
+
+Here at Adello, we do not only care how our creatives impact the publisher's overall page performance. We are also very aware that what helps the publisher, helps us. Two main points would be load speed and seamless integration.
+So to ensure that our creatives load seamlessly in their environment, here are a few rules of thumb we follow that are mainly categorized into two aspects, best practices and improvements.
+
+### Vanilla Code
+It is tempting to jump over using online libraries to get things done quicker and easier, such as jQuery, Bootstrap, GreenSock to name a few. However, if they are not heavily used, such as in a full-featured website, then they add too much overhead and impact load and browsing performance.<br>
+Taking [jQuery](https://jquery.com/) as example, nowadays, JavaScript is better supported in most browsers and we no longer have to cover so much cross-browser compatibility solutions. Already, a wide number of jQuery methods are supported natively by JavaScript and I don't need to say this, but native is better, and faster!
+Here are just a few common examples where JavaScript shines over methods that were popularly used with jQuery:
+
+```javascript
+$('.myClassName');     /* vs */  document.querySelector('.myClassName');
+$(parent).append(el);  /* vs */  parent.appendChild(el);
+$(el).clone();         /* vs */  el.cloneNode(true);
+$(el).outerHeight();   /* vs */  el.offsetHeight
+$(el).offset();        /* vs */  el.getBoundingClientRect();
+$.proxy(fn, context);  /* vs */  fn.bind(context);
+```
+
+When using a popular library, there is also the concern of version conflict with the one on the publisher. 
+I will stress a bit that jQuery and other similar libraries and frameworks still have strong uses, however, creatives are lighter in their nature and we need to weigh the Use versus Overhead.
```

Comment:
Why capitalize Use and Overhead?

Reply:
I guess that at the time of writing, I wanted to make sure they are not confused as verbs or as a typo, but that does not add much value so I removed the capitalization